contracts/router/src/contract/migrations/v1_0_0.rs

```diff
@@ -0,0 +1,121 @@
+use axelar_wasm_std::error::ContractError;
+use cosmwasm_std::{Addr, Storage};
+
+use crate::contract::CONTRACT_NAME;
+use crate::state;
+
+const BASE_VERSION: &str = "1.0.0";
+
+pub fn migrate(storage: &mut dyn Storage, axelarnet_gateway: Addr) -> Result<(), ContractError> {
+    cw2::assert_contract_version(storage, CONTRACT_NAME, BASE_VERSION)?;
```

Comment:
we had issued a v1.0.1 patch to the router. Will that cause an issue here since the base version isn't v1.0.0 on some networks I think? cc @eguajardo

Reply:
I don't know the release action was not on main branch.

@eguajardo can you confirm if all contracts deployed across networks are on 1.0.1?

Reply:
Router devnet-amplifier is on 1.0.1, all others (stagenet, testnet and mainnet) Router is 1.0.0. What version are we migrating to? If the migration is for 1.0.1 to 1.0.2, then BASE_VERSION should be set to 1.0.1 and all contracts must be migrated sequentially 1.0.0 -> 1.0.1 -> 1.0.2